cmd/local/local.go

```diff
@@ -168,18 +175,9 @@ func runLocal(cmd *cobra.Command, args []string) error {
 	progressPrinter.IncrementTracker("step-base", 1)
 	progressPrinter.IncrementTracker("step-github", 1)
 
-	//
-	// create local certs using MkCert tool
-	//
-	log.Info().Msg("installing CA from MkCert")
-	ssl.InstallCALocal(config)
-	log.Info().Msg("installing CA from MkCert done")
-
-	log.Info().Msg("creating local certificates")
-	if err := ssl.CreateCertificatesForLocalWrapper(config); err != nil {
+	if err := ssl.CreateCertificatesForLocalWrapper(config, disableTLS); err != nil {
```

Comment:
```suggestion
	if err := ssl.CreateCertificatesForLocalWrapper(config, disableTLS); err != nil {
```
curious about passing `config` as a parameter to the function, i've seen it a few times now but i mostly see something like [line#96](https://github.com/kubefirst/kubefirst/blob/081fa7bee82c57447b1ea09c3ee2db8b5de6766c/cmd/local/local.go#L96). do we need the entire config object or do we need a property from config? i want to understand so my incoming changes can follow best practices

Reply:
We could only pass the parameters that we cant, and avoid passing the whole config object, but to be honest, it's passing just the reference address in the memory where config object is since we're using pointers. In terms of performance might not be an issue.

for readability, I personally think just passing the only parameter we need would make more sense. I'll update to make it easier to read. good question! thanks!

Reply:
thanks for the details! i guess the answer is "it depends" 😄  
i feel like im taking away these two soft rules
 - if we need a single parameter pass `config.PARAM` (probably)
 - if we need multiple params from that object then we should pass the entire object? 
 is that reasonable?

Reply:
the senior answer "depends" 😄

that makes total sense for me.

to illustrate:

`func example(itemA string, itemB string, itemC string, itemD string) error {`

in this case, passing `config` help us to make the function signature short:

`func example(config *configs.Config) error {`